### PR TITLE
Change fieldType of uptime and nknc support export privKey

### PR DESCRIPTION
**Pull request by gdmmx**
Signed-off-by: gdmmx <gdmmx@163.com>

### Proposed changes in this pull request
Enhance nknc functionality, support export privateKey by verbose flag
Adjust fieldType of uptime from string to int

### Type (put an `x` where ever applicable)
- [ ] Bug fix: Link to the issue
- [x] Feature (Non-breaking change)
- [ ] Feature (Breaking change)
- [ ] Documentation Improvement

### Checklist
Please put an `x` against the checkboxes. Write a small comment explaining if its `N/A` (not applicable)

- [x] Read the [CONTRIBUTION guidelines](https://github.com/nknorg/nkn#contributing).
- [x] All the tests are passing after the introduction of new changes.
- [ ] Added tests respective to the part of code I have written.
- [ ] Added proper documentation where ever applicable (in code and README.md).
- [x] Code has been written according to [NKN-Golang-Style-Guide](https://github.com/nknorg/nkn/wiki/NKN-Golang-Style-Guide)

### Extra information
Any extra information related to this pull request.

### CODE REVIEW

File: cli/wallet/wallet.go [hunk 1]
```diff
@@ -96,9 +100,13 @@ func walletAction(c *cli.Context) error {
 				fmt.Fprintln(os.Stderr, err)
 				os.Exit(1)
 			}
+			var vbs bool
```
Comment (dreamfly281) [inline cli/wallet/wallet.go]: What is the vbs used to and how to protect the private key be show from remotely? I mean remote connect the nknd with nknc --ip --port

File: net/node/localnode.go [hunk 2]
```diff
@@ -63,7 +63,7 @@ func (localNode *LocalNode) MarshalJSON() ([]byte, error) {
 	}
 
 	out["height"] = localNode.GetHeight()
-	out["uptime"] = time.Since(localNode.startTime).Truncate(time.Second).String()
+	out["uptime"] = time.Since(localNode.startTime).Truncate(time.Second).Seconds()
```
Comment (dreamfly281) [inline net/node/localnode.go]: Any incompatible from client or SDK side?

File: cli/wallet/wallet.go (hunk 1 above)
Comment (yilunzhang) [reply cli/wallet/wallet.go]: nknc wallet always use local wallet.dat, --ip will be ignored

File: net/node/localnode.go (hunk 2 above)
Comment (yilunzhang) [reply net/node/localnode.go]: Nope, no one is using it atm

**Pull request merged**